qa/integration-tests/src/test/java/io/camunda/it/tasklist/compatibility/CompatibilityTasklistUserTaskAuthorizationIT.java

```diff
@@ -42,14 +42,18 @@ public class CompatibilityTasklistAssignUserTaskAuthorizationIT {
           .withBasicAuth()
           .withProperty("camunda.tasklist.zeebe.compatibility.enabled", true);
 
-  private static final String PROCESS_ID = "foo";
+  private static final String PROCESS_WITH_USER_TASK = "PROCESS_WITH_USER_TASK";
+  private static final String PROCESS_WITH_USER_TASK_PRE_ASSIGNED =
+      "PROCESS_WITH_USER_TASK_PRE_ASSIGNED";
   private static final String PROCESS_ID_WITH_JOB_BASED_USERTASK =
       "PROCESS_WITH_JOB_BASED_USERTASK";
+  private static final String PROCESS_ID_WITH_JOB_BASED_USERTASK_PRE_ASSIGNED =
+      "PROCESS_WITH_JOB_BASED_USERTASK_PRE_ASSIGNED";
 
   private static final String ADMIN_USER_NAME = "foo";
   private static final String ADMIN_USER_PASSWORD = "foo";
 
-  private static final String TEST_USER_NAME_NO_PERMISSION = "noPermissionUser";
```

Comment:
ℹ️ We first had a better name, but this user name is used in processes, for assignments. Renaming this causes more issues with other tests, so I kept bar for now. I will refactor this later.